Source/dapi/Server.cpp

```diff
@@ -2401,6 +2401,7 @@ void Server::saveGame()
 	if (devilution::gbIsMultiplayer || !devilution::gmenu_is_active())
 		return;
 
+	devilution::gmenu_presskeys(SDLK_DOWN);
 	devilution::gmenu_presskeys(SDLK_KP_ENTER);
 	return;

```

Comment:
DevilutionX has quick save:
```cpp
if (devilution::gbIsMultiplayer)
		return;
gamemenu_save_game(false);
```

Reply:
Does it make the menu sounds though? I could just add a menu sound in like I did for 1.09 though. 😆

Reply:
The human player can do this, it's bound to one of the F-keys by default.

It shows the notification, but no it doesn't play the sound.